database/fingertips-db/PostDeployment/PopulateFingertips.sql

```diff
@@ -313,9 +338,136 @@ INSERT [dbo].[SexDimension] ([SexKey], [Name], [IsFemale], [HasValue], [SexId])
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4471, 68430, 3334, 1, 611, 88886, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4472, 68430, 3334, 1, 612, 78706, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4473, 68430, 3334, 1, 613, 66175, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4474, 68430, 3334, 1, 614, 58820, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4475, 68430, 3334, 1, 615, 51932, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4476, 68430, 3334, 1, 616, 42323, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4477, 68430, 3334, 1, 617, 30839, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4478, 68430, 3334, 1, 618, 22558, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4479, 68430, 3334, 1, 619, 18231, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4480, 68430, 3334, 1, 620, 11208, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4481, 68430, 3334, 1, 621, 6759, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4482, 68430, 3334, 1, 630, 2809, 0, 0, 0, 2024)
+INSERT [dbo].[HealthMeasure] ([HealthMeasureKey], [AreaKey], [IndicatorKey], [SexKey], [AgeKey], [Count], [Value], [LowerCI], [UpperCI], [Year]) VALUES (4483, 68430, 3334, 1, 631, 709, 0, 0, 0, 2024)
+
+SET IDENTITY_INSERT [dbo].[HealthMeasure] OFF
+
+
+--create some health measure data
+SET IDENTITY_INSERT [dbo].[HealthMeasure] ON SET IDENTITY_INSERT [dbo].[HealthMeasure] ON 
```

Comment:
I think you've duplicated this statement
```suggestion
SET IDENTITY_INSERT [dbo].[HealthMeasure] ON 
```